
<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

   <meta property="og:url" content="https://plantgeneticslab.github.io/publication/" />
   <link rel="canonical" href="https://plantgeneticslab.github.io/publication/" />


  <link rel="shortcut icon" href="/favicon.ico?" type="image/x-icon">
<link rel="icon" href="home/favicon.ico?" type="image/x-icon">

    <meta name="description" content="Lu Lab at Institute of Genetics and Developmental Biology, Chinese Academy of Sciences">

    <title>PUBLICATION - Plant Genetics Lab</title>

    <link rel="canonical" href="https://plantgeneticslab.github.io/home/publication/">

<!-- CSS -->
	
    <!-- Bootstrap Core CSS -->
    <link rel="stylesheet" href="home/css/bootstrap.min.css">
    <!--<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">-->
    <!--<link rel="stylesheet" href="https://cdn.staticfile.org/twitter-bootstrap/4.3.1/css/bootstrap.min.css">-->

    <!-- Custom CSS -->
    <link rel="stylesheet" href="home/css/main.css">
    <link rel="stylesheet" href="home/css/galleries.css">
    <link rel="stylesheet" href="home/css/magnificpopup.css">

    <!-- Custom Fonts -->
    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Josefin+Sans:400,600&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
    
    <!--移动设备优先-->
    <!--<meta name="viewport" content="width=device-width, initial-scale=1.0">-->

</head>


<head>

<meta name="viewport" content="width=device-width,initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=yes"/>

</head>

<body>

    <!-- Navigation -->
<nav class="navbar navbar-default navbar-custom navbar-fixed-top" style="margin-right:1em; z-index: 1;">
  <div class="container-fluid">
      <!-- Brand and toggle get grouped for better mobile display -->
      <div class="navbar-header page-scroll">
          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
              <i class="fa fa-bars fa-lg"></i>
          </button>
          <a href="index.html" title="HOME"><img src="home/logo.png" alt="logo" style="width:150px;cursor:pointer;margin-top:15px;"/></a>
      </div>
<style>
        .drop-down-content{
            padding: 0px;
            display: none;
            list-style: none;
            overflow-y:hidden;
            line-height: 5px;
            justify-content: center;
        }
       .drop-down:hover .drop-down-content{
            display:block;
            right:0px;
            max-width: width 700px; ;
        }
        .drop-down {
            padding: 0px;
        }
        .m{
           cursor: pointer;
        }
        .dropdown:hover > dropdown-menu {
        display: flex;
        margin: 0 0 0;}
  </style>   

      <div class="collapse navbar-collapse m" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav navbar-right" style="margin-right:0em; text-shadow: -.025em -.025em 0 rgba(0, 133, 161, 0.75) ">
      <!--<ul class="nav navbar-nav"></ul>-->
       <li><a href="about.html">About</a></li>
        <li><a href="research.html">Research</a></li>
        <li class="drop-down">
        <a href="team.html" class="drop-down dropdown" >People</a>
          <ul  class="drop-down-content">
            <li><a href="teamalbum.html">Album</a></li>
          </ul>
        </li>
        <li><a href="publication.html">Publication</a></li>

        <li class="drop-down">
          <a href="software.html" style="width: 179px;">Software</a>
          <ul class="drop-down-content">
            <li><a href="tiger.html">Tiger</a></li>
            <li><a href="SiPAS.html">SiPAS-tools</a></li>
          </ul>
        </li>

        <li>
          <a href="resource.html">Resource</a> 
          <ul class="drop-down-content">
            <li><a href="Vmap.html">VMap</a></li>
          </ul>
        </li>

        <li class="drop-down">
          <a href="roundtable.html">RoundTable</a>
          <ul class="drop-down-content">
            <li><a href="journal.html">Journal</a></li>
            <li><a href="forum.html">Forum</a></li>
          </ul>
        </li>

        <li><a href="news.html">News</a></li>
        <li><a href="jobs.html">Jobs</a></li>

       

      </ul>
      </div>
      
      <!-- Collect the nav links, forms, and other content for toggling -->
      <!--
      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
        <ul class="nav navbar-nav navbar-right" style="margin-right:1em; text-shadow: -.025em -.025em 0 rgba(0, 133, 161, 0.75), .025em -.025em 0 rgba(0, 133, 161, 0.75), -.025em .025em 0 rgba(0, 133, 161, 0.75), .025em .025em 0 rgba(0, 133, 161, 0.75);">
        -->     
      
      <!-- /.navbar-collapse -->
  </div>
  <!-- /.container -->
</nav>


    <!-- Page Header -->
<header class="intro-header" style="background-image: url('home/images/paper.jpg')">
    <meta name="viewport" content="width=device-width,initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=yes"/>
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <div class="site-heading">
                    <h1>PUBLICATION</h1>
                    <!--<hr class="small">-->
                    <span class="subheading">Plant Genetics Lab</span>
                </div>
            </div>
        </div>
    </div>
</header>

<!-- Main Content -->
<div class="container">
	<div class="row">
					<style>
.xie{
    font-weight: bold;
    font-style: oblique;   
}
.xie2{
    font-style: oblique;
}
.bold{
    font-weight: bold;
}
.subhead{
    color: darkgrey;
}
a,a:hover{
    color:skyblue;
}
a,a:focus{
    color: orange;
}

</style>
<h2><a href="https://scholar.google.com/citations?user=RbNQWbUAAAAJ&hl=en"><bold>Google Scholar</bold></a></h2>
<br>

<h2>SELECTED PUBLICATIONS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>

<p>
    6. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Changbin Yin, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, 
    Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Zhiyong Liu, Xiangdong Fu, 
    <span class="bold">Fei Lu</span>. Population genomics unravels the Holocene history of bread wheat and its relatives.
    <span class="xie">Nature Plants. </span>(2023). 
    <br><a href=" https://doi.org/10.1038/s41477-023-01367-3">doi:10.1038/s41477-023-01367-3</a>

</p>

<p>
    5. Yuanying Peng#,*, Honghai Yan#, Laichun Guo#, Cao Deng#, Chunlong Wang#, Yubo Wang#, Lipeng Kang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Xiaomeng Liu, Zongyi Sun, 
    Yun Peng, Jun Zhao, Di Deng, Yinghong Xu, Ying Li, Qiantao Jiang, Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Zhengsong Peng, Dengcai Liu, 
    Jizeng Jia, Youliang Zheng, Tao Ma*, Yuming Wei*, <span class="bold">Fei Lu*</span>, Changzhong Ren*. 
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat.
    <span class="xie">Nature Genetics. </span>(2022). 
    <br><a href="https://www.nature.com/articles/s41588-022-01127-7">doi:10.1038/s41588-022-01127-7.</a>
</p>

<p>
    4. Yao Zhou#, Xuebo Zhao#, Yiwen Li, Jun Xu, Aoyue Bi, Lipeng Kang, Daxing Xu, Haofeng Chen, Ying Wang, Yuan-ge Wang, Sanyang Liu, Chengzhi Jiao, Hongfeng Lu, Jing Wang, Changbin Yin, Yuling Jiao*, <span class="bold">F. Lu*</span>.
    <span class="xie2">Triticum</span> population sequencing provides insights into wheat adaptation. 
    <span class="xie">Nature Genetics. </span>(2020). 
    <br><a href="https://www.nature.com/articles/s41588-020-00722-w">doi:10.1038/s41588-020-00722-w.</a>
</p>

<p>
    3. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>.
    Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation. 
    <span class="xie">Nature Genetics. </span>(2017).
   <br> <a href="https://www.nature.com/articles/ng.3845">doi:10.1038/ng.3845</a>
</p>

<p>
    2. <span class="bold">Fei Lu*</span>, Maria C Romay, Jeff C Glaubitz, Peter J Bradbury, Rob J Elshire, Tianyu Wang, Yu Li, Yongxiang Li, Kassa Semagn, Xuecai Zhang, Alvaro G. Hernandez, Mark A. Mikel, Ilya Soifer, Omer Barad, Edward S Buckler*. 
    High-resolution genetic mapping of maize pan-genome sequence anchors. 
    <span class="xie">Nature Communications. </span>(2015).
    <br><a href="https://www.nature.com/articles/ncomms7914 ">doi: 10.1038/ncomms7914. </a>
</p>

<p>
   1.<span class="bold"> Lu F</span>, Lipka AE, Elshire RJ, Glaubitz JC, Cherney JH, Casler MD, Buckler ES, Costich DE.
    Switchgrass genomic diversity, ploidy and evolution: novel insights from a network-based SNP discovery protocol. 
    <span class="xie"> PLoS Genetics. </span>(2013).
    <br><a href="https://doi.org/10.1371/journal.pgen.1003215">https://doi.org/10.1371/journal.pgen.1003215</a>
    (Citation: <span class="bold">675</span> by 2022-07-14)
</p>

<br>

<h2>PREPRINTS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>

<p>
    8. Aoyue Bi#, Daxing Xu#, Lipeng Kang, Yafei Guo, Xinyue Song, Xuebo Zhao, Jijin Zhang, Zhiliang Zhang, Yiwen Li, Changbin Yin, Jing Wang, 
    <span class="bold">Fei Lu</span>. An integrated map of genetic variation from 1,062 wheat genomes. 
    <span class="xie">bioRxiv. </span>(2023).
    <br><a href="https://www.biorxiv.org/content/10.1101/2023.03.31.535022v1"> doi:10.1101/2023.03.31.535022</a>
</p>    

<p>
    7. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Changbin Yin, Zhiyong Liu, Xiangdong Fu, <span class="bold">Fei Lu*</span>.
    Population genomics unravels the Holocene history of <span class="xie2">Triticum-Aegilops</span> species.
    <span class="xie">bioRxiv. </span>(2022).
    <br><a href="https://www.biorxiv.org/content/10.1101/2022.04.07.487499v1"> doi:10.1101/2022.04.07.487499</a>
</p>

<p>
    6. zhiliang Zhang#, Jijin Zhang#, Lipeng Kang, Xuebing Qiu, Beirui Niu, Aoyue Bi, Xuebo Zhao, Daxing Xu, Jing Wang, Changbin Yin, Xiangdong Fu, <span class="bold">Fei Lu*</span>.
    Genotyping of structural variation using PacBio high-fidelity sequencing.
    <span class="xie">bioRxiv. </span>(2021).
    <br><a href="https://www.biorxiv.org/content/10.1101/2021.10.28.466362v1.full">doi:10.1101/2021.10.28.466362</a>
</p>

<p>
    5. Yuanying Peng*, Honghai Yan, Laichun Guo, Cao Deng, Lipeng Kang, Chunlong Wang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Jun Zhao, Yun Peng, Xiaomeng Liu, Di Deng, 
    Yinghong Xu, Ying Li, Qiantao Jiang, Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Youliang Zheng, Yuming Wei, <span class="bold">F. Lu*</span>, Changzhong <Ren class="">
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat. 
    <span class="xie">Research Square. </span> (2021).
    <br><a href="https://europepmc.org/article/ppr/ppr363254"> doi:10.21203/rs.3.rs-664692/v1</a>
</p>

<p>
    4. X. Zhao, X. Fu, C. Yin*,  <span class="bold">F. Lu*</span>.Wheat Speciation and Adaptation: Perspectives from Reticulate Evolution. 
    <span class="xie">Preprints. </span>(2021).
    <br><a href="https://doi.org/10.1007/s42994-021-00047-0">https://doi.org/10.1007/s42994-021-00047-0</a>
</p>

<p>
    3. Y. Zhou#, X. Zhao#, Y. Li, J. Xu, A. Bi, L. Kang, H. Chen, Y. Wang, Y. Wang, S. Liu, C. Jiao, H. Lu, J. Wang, C. Yin, Y. Jiao*,<span class="bold">F. Lu*</span>,
    Convergence within divergence: insights of wheat adaptation from <span class="xie2">Triticum </span> sequencing.
    <span class="xie">bioRxiv. </span>(2020).
    <br><a href="https://www.biorxiv.org/content/10.1101/2020.03.21.001362v1">doi:10.1101/2020.03.21.001362</a>
</p>

<p>
    2. H. Chen, C. Jiao, Y. Wang, Y. Wang, C. Tian, H. Yu, J. Wang, X. Wang, <span class="bold">F. Lu</span>, X. Fu, Y. Xue, W. Jiang, H. Ling, H. Lu, Y. Jiao. 
    Comparative Population Genomics of Bread Wheat (<span class="xie2">Triticum aestivum</span>) Reveals Its Cultivation and Breeding History in China.
    <span class="xie">bioRxiv. </span>(2019).
    <br><a href="https://doi.org/10.1101/519587">doi:10.1101/519587</a>
</p>


<p>
    1. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>.
    Cassava HapMap: Managing genetic load in a clonal crop species. 
    <span class="xie">bioRxiv. </span>(2016).
    <br><a href="https://www.biorxiv.org/content/10.1101/077123v1">doi:http://dx.doi.org/10.1101/077123</a>
</p>


<br>
<h2>PEER-REVIEWED PUBLICATIONS</h2>
<h3>(*Corresponding author; #Co-first author)</h3>

<p>
    18. Yangyang Liu#, Kuocheng Shen#, Changbin Yin#, Xiaowan Xu#, Xuchang Yu, Botao Ye, Zhiwen Sun, Jiayu Dong, Aoyue Bi, 
    Xuebo Zhao, Daxing Xu, Zhonghu He, Xueyong Zhang, Chenyang Hao, Jianhui Wu6, Ziying Wang, He Wu, Danni Liu, Lili Zhang, 
    Liping Shen, Yuanfeng Hao*, <span class="bold">Fei Lu*</span>, Zifeng Guo*. Genetic basis of geographical differentiation and breeding selection for wheat plant architecture traits. 
    <span class="xie">Genome Biology. </span>(2023).
    <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02932-x">doi:10.1186/s13059-023-02932-x</a>
</p>    
<p>
    17. Xuebo Zhao#, Yafei Guo#, Lipeng Kang, Changbin Yin, Aoyue Bi, Daxing Xu, Zhiliang Zhang, Zhang Jijin, 
    Xiaohan Yang, Jun Xu, Song Xu, Xinyue Song, Ming Zhang, Yiwen Li, Philip Kear, Jing Wang, Zhiyong Liu, Xiangdong Fu, 
    <span class="bold">Fei Lu</span>. Population genomics unravels the Holocene history of bread wheat and its relatives.
    <span class="xie">Nature Plants. </span>(2023). 
    <br><a href=" https://doi.org/10.1038/s41477-023-01367-3">doi:10.1038/s41477-023-01367-3</a>
</p>
<p>
    16. Hao Jiang, Yu Fang, Dong Yan, Si-tong Liu, Jun Wei, Fei-long Guo, Xing-ting Wu, Hong Cao, Chang-bin Yin, 
    <span class="bold"> Fei Lu</span>, Li-feng Gao & Yong-xiu Liu. (2022).
    Genome-wide association study reveals a NAC transcription factor TaNAC074 linked to pre-harvest sprouting tolerance in wheat. 
    <span class="xie">Theor. Appl. Genet. </span> (2022). 
    <br><a href="https://doi.org/10.1007/s00122-022-04184-y">doi:10.1007/s00122-022-04184-y.</a>
</p>

<p>
    15. Jun Xiao, Bao Liu, Yingyin Yao, Zifeng Guo, Haiyan Jia, Lingrang Kong, Aimin Zhang, Wujun Ma, Zhongfu Ni, Shengbao Xu, <span class="bold"> Fei Lu</span>, 
    Yuannian Jiao, Wuyun Yang, Xuelei Lin, Silong Sun, Zefu Lu, Lifeng Gao, Guangyao Zhao, Shuanghe Cao, Qian Chen, Kunpu Zhang, Mengcheng Wang, Meng Wang, 
    Zhaorong Hu, Weilong Guo, Guoqiang Li, Xin Ma, Junming Li, Fangpu Han, Xiangdong Fu, Zhengqiang Ma, Daowen Wang, Xueyong Zhang, Hong-Qing Ling, Guangmin Xia, 
    Yiping Tong, Zhiyong Liu, Zhonghu He, Jizeng Jia & Kang Chong.
    Wheat genomic study for genetic improvement of traits in China.
    <span class="xie">Sci. China Life Sci. </span> (2022). 
    <br><a href="https://doi.org/10.1007/s11427-022-2178-7">doi:10.1007/s11427-022-2178-7.</a>
</p>

<p>
    14. Yuanying Peng#,*, Honghai Yan#, Laichun Guo#, Cao Deng#, Chunlong Wang#, Yubo Wang#, Lipeng Kang, Pingping Zhou, Kaiquan Yu, Xiaolong Dong, Xiaomeng Liu, Zongyi Sun, Yun Peng, Jun Zhao, Di Deng, Yinghong Xu, Ying Li, Qiantao Jiang, 
    Yan Li, Liming Wei, Jirui Wang, Jian Ma, Ming Hao, Wei Li, Houyang Kang, Zhengsong Peng, Dengcai Liu, Jizeng Jia, Youliang Zheng, Tao Ma*, Yuming Wei*, <span class="bold">Fei Lu*, </span>Changzhong Ren*.
    Reference genome assemblies reveal the origin and evolution of allohexaploid oat. 
    <span class="xie">Nature Genetics. </span> (2022).
    <br><a href="https://www.nature.com/articles/s41588-022-01127-7">doi:10.1038/s41588-022-01127-7.</a>
</p>

<p>
    13. Yuange Wang#, Fei Du#, Jian Wang, Ke Wang, Caihuan Tian, Xiaoquan Qi,<span class="bold"> Fei Lu</span>, Xigang Liu, Xingguo Ye, and Yuling Jiao*. 
    Improving bread wheat yield through modulating an unselected AP2/ERF gene. 
    <span class="xie">Nature Plants. </span> (2022).
    <br><a href="https://www.nature.com/articles/s41477-022-01197-9">doi:10.1038/s41477-022-01197-9.</a>
</p>

<p>
    12. Xiaofei Yang, Haopeng Yu, Wenqing Sun, Ling Ding, Ji Li, Jitender Cheema, Ricardo Ramirez-Gonzalez, Xuebo Zhao, Azahara C. Martín, <span class="bold">Fei Lu, </span>Bao Liu, Cristobal Uauy, Yiliang Ding & Huakun Zhang. 
    Wheat in vivo RNA structure landscape reveals a prevalent role of RNA structure in modulating translational subgenome expression asymmetry. 
    <span class="xie">Genome Biology. </span>(2021).
    <br><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02549-y">doi: 10.1186/s13059-021-02549-y</a>
</p>

<p>
    11. Zhang, X.*, Zhu, Y., Kremling, K.A.G., Romay, M.C., Bukowski, R., Sun, Q., Gao, S., Buckler, E.S., <span class="bold"> Lu, F.*.</span> 
    Genome‑wide analysis of deletions in maize population reveals abundant genetic diversity and functional impact. 
    <span class="xie">Theor. Appl. Genet. </span> (2021).
    <br><a href="https://link.springer.com/article/10.1007/s00122-021-03965-1">doi:10.1007/s00122-021-03965-1</a>
</p>

<p>
    10. Wang J#, Xu J#, Yang X#, Xu S, Zhang M, <span class="bold">F. Lu*</span>.
    Boosting the power of transcriptomics by developing an efficient gene expression profiling approach.
    <span class="xie"> Plant Biotechnol J. </span> (2021)
    <br><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/pbi.13706">doi:10.1111/PBI.13706</a>
</p>


<p>
    9. X. Zhao, X. Fu, C. Yin*, <span class="bold">F. Lu*</span>.Wheat Speciation and Adaptation: Perspectives from Reticulate Evolution. 
    <span class="xie">aBiotech.</span> (2021)
    <br><a href="https://link.springer.com/article/10.1007/s42994-021-00047-0"> doi:10.1007/s42994-021-00047-0</a>
</p>


<p>
    8. H. Zhai, C. Jiang, Y. Zhao, S. Yang, Y. Li, K. Yan, S. Wu, B. Luo, Y. Du, H. Jin, X. Liu, Y. Zhang, <span class="bold">F. Lu</span>, 
    M. Reynolds, X. Ou, W. Qiao, Z. Jiang, T. Peng, D. Gao, W. Hu, J. Wang, H. Gao, G. Yin, K. Zhang, G. Li, D. Wang, 
    Wheat heat tolerance is impaired by heightened deletions in the distal end of 4AL chromosomal arm. 
    <span class="xie">Plant Biotechnol. J. </span> (2020).
    <br><a href="https://onlinelibrary.wiley.com/doi/10.1111/pbi.13529">doi: https://doi.org/10.1111/pbi.13529</a>
</p>

<p>
    7. Y. Zhou#, X. Zhao#, Y. Li, J. Xu, A. Bi, L. Kang, D. Xu, H. Chen, Y. Wang, Y. Wang, S. Liu, C. Jiao, H. Lu, J. Wang, C. Yin, Y. Jiao*, <span class="bold">F. Lu*.</span>
    <span class="xie2">Triticum </span> sequencing provides insights into wheat adaptation. 
    <span class="xie"> Nature Genetics. </span> (2020). 
    <br><a href="https://www.nature.com/articles/s41588-020-00722-w">doi:10.1038/s41588-020-00722-w</a>
</p>

<p>
    6. Chunzhi Zhang, Pei Wang, Die Tang, Zhongmin Yang, <span class="bold">Fei Lu</span>, Jianjian Qi, Nilesh R. Tawari, Yi Shang, Canhui Li & Sanwen Huang. 
    The genetic basis of inbreeding depression in potato.
    <span class="xie"> Nature Genetics. </span> (2019). 
    <br><a href="https://www.nature.com/articles/s41588-018-0319-1">doi:10.1038/s41588-018-0319-1</a>
</p>

<p>
    5. Lian, Q., Tang, D., Bai, Z., Qi, J., <span class="bold">Lu, F.</span>, Huang, S., Zhang, C. 
    Acquisition of deleterious mutations during potato polyploidization.
    <span class="xie"> J Integr Plant Biol. </span> (2019). 
    <br><a href="https://onlinelibrary.wiley.com/doi/10.1111/jipb.12748"> https://doi.org/10.1111/jipb.12748</a>
</p>

<p>
    4. Springer Nathan M., Anderson Sarah N., Andorf Carson M., Ahern Kevin R., Bai Fang, Barad Omer, Barbazuk W. Brad, Bass Hank W., Baruch Kobi, Ben-Zvi Gil, Buckler Edward S.,
     Bukowski Robert, Campbell Michael S., Cannon Ethalinda K. S., Chomet Paul, Dawe R. Kelly, Davenport Ruth, Dooner Hugo K., Du Limei He, Du Chunguang, Easterling Katherine A.,
      Gault Christine, Guan Jiahn-Chou, Hunter Charles T., Jander Georg, Jiao Yinping, Koch Karen E., Kol Guy, Köllner Tobias G., Kudo Toru, Li Qing, <span class="bold">Lu Fei</span>, 
      Mayfield-Jones Dustin, Mei Wenbin, McCarty Donald R., Noshay Jaclyn M., Portwood John L., Ronen Gil, Settles A. Mark, Shem-Tov Doron, Shi Jinghua, Soifer, Ilya, Stein Joshua C., 
      Stitzer Michelle C., Suzuki Masaharu, Vera Daniel L., Vollbrecht Erik, Vrebalov Julia T., Ware Doreen, Wei Sharon, Wimalanathan, Kokulapalan, Woodhouse Margaret R., Xiong Wenwei, Brutnell Thomas P. 
      The maize W22 genome provides a foundation for functional genomics and transposon biology. 
      <span class="xie"> Nature Genetics. </span> (2018).
      <br><a href="https://www.nature.com/articles/s41588-018-0158-0 ">doi:10.1038/s41588-018-0158-0 </a>
</p>

<p>
    3. Sun Silong, Zhou Yingsi, Chen Jian, Shi Junpeng, Zhao Haiming, Zhao Hainan, Song Weibin, Zhang Mei, Cui Yang, Dong Xiaomei, Liu Han, 
    Ma Xuxu, Jiao Yinping, Wang Bo, Wei Xuehong, Stein Joshua C., Glaubitz Jeff C., <span class="bold">Lu Fei</span>, Yu Guoliang, 
    Liang Chengzhi, Fengler Kevin, Li Bailin, Rafalski Antoni, Schnable Patrick S., Ware Doreen H., Buckler Edward S., Lai Jinsheng. 
    Extensive intraspecific gene order and gene structural variations between Mo17 and other maize genomes.
    <span class="xie"> Nature Genetics. </span> (2018).
    <br><a href="https://doi.org/10.1038/s41588-018-0182-0">doi:10.1038/s41588-018-0182</a>
</p>

<p>
    2. Karl A. G. Kremling, Shu-Yun Chen, Mei-Hsiu Su, Nicholas K. Lepak, M. Cinta Romay, Kelly L. Swarts, <span class="bold">Fei Lu</span>, Anne Lorant, Peter J. Bradbury & Edward S. Buckler. 
    Dysregulation of expression correlates with rare-allele burden and fitness loss in maize. 
    <span class="xie"> Nature. </span> (2018).
    <br><a href="https://www.nature.com/articles/nature25966">doi:10.1038/nature25966</a>
</p>

<p>
    1. Punna Ramu*, Williams Esuma, Robert Kawuki, Ismail Y Rabbi, Chiedozie Egesi, Jessen V Bredeson, Rebecca S Bart, Janu Verma, Edward S Buckler, <span class="bold">Fei Lu*</span>. 
    Cassava haplotype map highlights fixation of deleterious mutations during clonal propagation. 
    <span class="xie"> Nature Genetics. </span> (2017).
    <br><a href="https://www.nature.com/articles/ng.3845">doi:10.1038/ng.3845</a>
</p>
<br>
<br>
<br>
<br>

	</div>
</div>

<!-- jQuery -->   

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js"></script>
<!--
<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
  <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
-->

<!-- Custom Theme JavaScript -->
<!--<script src="/home/js/main.min.js "></script>-->

<!-- Default Statcounter code for LuLab
https://plantgeneticslab.github.io/home/ -->
<script type="text/javascript">
var sc_project=12732004; 
var sc_invisible=1; 
var sc_security="d1536717"; 
</script>
<script type="text/javascript"
src="https://www.statcounter.com/counter/counter.js"
async></script>
<noscript><div class="statcounter"><a title="web counter"
href="https://statcounter.com/" target="_blank"><img
class="statcounter"
src="https://c.statcounter.com/12732004/0/d1536717/1/"
alt="web counter"
referrerPolicy="no-referrer-when-downgrade"></a></div></noscript>
<!-- End of Statcounter Code -->

    <!--<a href="javascript:void(0)" title="SEARCH" onclick="superSearch.toggle()" class="super-search-btn">
<span class="fa fa-search" style="font-size:1em; position:fixed; right: .75em; top: 1em; z-index: 2;"></span>
</a>
<div class="super-search" id="js-super-search">
<a href="javascript:void(0)" onclick="superSearch.toggle()" class="super-search__close-btn">
<span class="fa fa-close" style="font-size:48px;"></span>
</a>
<input type="text" placeholder="Type here to search" class="super-search__input" id="js-super-search__input">
<ul class="super-search__results" id="js-super-search__results"></ul>
</div>



 <script src="/home/js/super-search.js"></script>

<script>
superSearch({
    searchFile: '/home/feed.xml',
    searchSelector: '#js-super-search', // CSS Selector for search container element.
    inputSelector: '#js-super-search__input', // CSS selector for <input>
    resultsSelector: '#js-super-search__results' // CSS selector for results container
});
</script>-->

    <!--
        <div class="row"  style="background-color: #530720; padding-bottom: 40px; margin-top: 50px; margin-bottom: 80px;">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">

      <center><h4><p>SUBSCRIBE TO RECEIVE POSTS DIRECTLY TO YOUR INBOX</h4></center>


<form style="padding:3px;text-align:center;font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size:12px;" action="https://tinyletter.com/photorgasms" method="post" target="popupwindow" onsubmit="window.open('https://tinyletter.com/yourproject', 'popupwindow', 'scrollbars=yes,width=800,height=600');return true"><input type="text" placeholder="Enter your email address"  style="width:200px;height:30px;border:0;background-color:#5A0F28;color:#dddddd;outline:0;padding-left:12px;" name="email" id="tlemail" />&nbsp; &nbsp; <input type="submit" value="Get notified!" style="background-color:#470118;color:#dddddd;height:30px;border:0;" /></form>
            </div>
        </div>
     
<style type="text/css">
        .tlemail::-webkit-input-placeholder {
   color: rgba(255,255,255,.45);
}
       .tlemail:-moz-placeholder { /* Firefox 18- */
   color: rgba(255,255,255,.45);  
}

      .tlemail::-moz-placeholder {  /* Firefox 19+ */
   color: rgba(255,255,255,.45);  
}

      .tlemail:-ms-input-placeholder {  
   color: rgba(255,255,255,.45);  
}
    </style>
-->

    <!-- Footer -->
<footer>
<div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">

                <ul class="list-inline text-center">

                    
                    <li>
                        <a href="https://twitter.com/fei_genetics" data-toggle="tooltip" title="Twiiter">
                            <span class="fa-stack fa-lg">
                                <i class="fa fa-circle fa-stack-2x"></i>
                                <i class="fa fa-twitter fa-stack-1x fa-inverse"></i>
                            </span>
                        </a>
                    </li>
                    
                    
                    
                    <li>
                        <a href="https://github.com/PlantGeneticsLab" data-toggle="tooltip" title="Github">
                            <span class="fa-stack fa-lg">
                                <i class="fa fa-circle fa-stack-2x"></i>
                                <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                            </span>
                        </a>
                    </li>
                    
            <li>
    <a href="mailto:dr.lufei@gmail.com" data-toggle="tooltip" target="_blank" title="Email">
                       <span class="fa-stack fa-lg">
                                <i class="fa fa-circle fa-stack-2x"></i>
                                <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
</span>
          </a> </li>
        <!--
         <li>
                        <a href="/home/feed.xml" data-toggle="tooltip" title="Feed">
                            <span class="fa-stack fa-lg">
                                <i class="fa fa-circle fa-stack-2x"></i>
                                <i class="fa fa-rss fa-stack-1x fa-inverse"></i>
                            </span>
                        </a>
                    </li>
                </ul>
        -->        
                <p class="copyright text-muted">Copyright &copy; Plant Genetics Lab 2023</p>
<!--<center><h6><p>Powered by <a href="https://github.com/jekyll/">Jekyll</a>.</p></h6></center>-->
            <br>
            </div>
        </div>
    </div>
</footer>


   <script>
//** jQuery Scroll to Top Control script- (c) 
//** v1.1 (April 7th, 10')
//** 1) Adds ability to scroll to an absolute position (from top of page) or specific element on the page instead.
//** 2) Fixes scroll animation not working in Opera. 
var scrolltotop={
//startline: Integer. Number of pixels from top of doc scrollbar is scrolled before showing control
//scrollto: Keyword (Integer, or "Scroll_to_Element_ID"). How far to scroll document up when control is clicked on (0=top).
setting: {startline:100, scrollto: 0, scrollduration:1000, fadeduration:[500, 100]},
controlHTML: '<img src="https://cloud.githubusercontent.com/assets/14811095/13691821/88412468-e744-11e5-8bb5-94340afd92e7.png" style="filter:alpha(opacity=100); -moz-opacity:1;"/>', //HTML for control, which is auto wrapped in DIV w/ ID="topcontrol"
controlattrs: {offsetx:35, offsety:60}, //offset of control relative to right/ bottom of window corner
anchorkeyword: '#top', //Enter href value of HTML anchors on the page that should also act as "Scroll Up" links
state: {isvisible:false, shouldvisible:false},
scrollup:function(){
if (!this.cssfixedsupport) //if control is positioned using JavaScript
this.$control.css({opacity:0}) //hide control immediately after clicking it
var dest=isNaN(this.setting.scrollto)? this.setting.scrollto : parseInt(this.setting.scrollto)
if (typeof dest=="string" && jQuery('#'+dest).length==1) //check element set by string exists
dest=jQuery('#'+dest).offset().top
else
dest=0
this.$body.animate({scrollTop: dest}, this.setting.scrollduration);
},
keepfixed:function(){
var $window=jQuery(window)
var controlx=$window.scrollLeft() + $window.width() - this.$control.width() - this.controlattrs.offsetx
var controly=$window.scrollTop() + $window.height() - this.$control.height() - this.controlattrs.offsety
this.$control.css({left:controlx+'px', top:controly+'px'})

},

togglecontrol:function(){
var scrolltop=jQuery(window).scrollTop()
if (!this.cssfixedsupport)
this.keepfixed()
this.state.shouldvisible=(scrolltop>=this.setting.startline)? true : false
if (this.state.shouldvisible && !this.state.isvisible){
this.$control.stop().animate({opacity:1}, this.setting.fadeduration[0])
this.state.isvisible=true
}
else if (this.state.shouldvisible==false && this.state.isvisible){
this.$control.stop().animate({opacity:0}, this.setting.fadeduration[1])
this.state.isvisible=false
}

},
init:function(){
jQuery(document).ready(function($){
var mainobj=scrolltotop
var iebrws=document.all

mainobj.cssfixedsupport=!iebrws || iebrws && document.compatMode=="CSS1Compat" && window.XMLHttpRequest //not IE or IE7+ browsers in standards mode
mainobj.$body=(window.opera)? (document.compatMode=="CSS1Compat"? $('html') : $('body')) : $('html,body')
mainobj.$control=$('<div id="topcontrol">'+mainobj.controlHTML+'</div>')
.css({position:mainobj.cssfixedsupport? 'fixed' : 'absolute', bottom:mainobj.controlattrs.offsety, right:mainobj.controlattrs.offsetx, opacity:0, cursor:'pointer'})
.attr({title:'Scroll To Top'})
.click(function(){mainobj.scrollup(); return false})
.appendTo('body')

if (document.all && !window.XMLHttpRequest && mainobj.$control.text()!='') //loose check for IE6 and below, plus whether control contains any text
mainobj.$control.css({width:mainobj.$control.width()}) //IE6- seems to require an explicit width on a DIV containing text
mainobj.togglecontrol()

$('a[href="' + mainobj.anchorkeyword +'"]').click(function(){
mainobj.scrollup()

return false
})

$(window).bind('scroll resize', function(e){

mainobj.togglecontrol()

})

})

}

}

scrolltotop.init()
</script>

</body>

<!-- Default Statcounter code for LuLab
https://plantgeneticslab.github.io/home/ -->
<script type="text/javascript">
var sc_project=12732004; 
var sc_invisible=1; 
var sc_security="d1536717"; 
</script>
<script type="text/javascript"
src="https://www.statcounter.com/counter/counter.js"
async></script>
<noscript><div class="statcounter"><a title="web counter"
href="https://statcounter.com/" target="_blank"><img
class="statcounter"
src="https://c.statcounter.com/12732004/0/d1536717/1/"
alt="web counter"
referrerPolicy="no-referrer-when-downgrade"></a></div></noscript>
<!-- End of Statcounter Code -->

</html>
